src/components/Book.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function Book(props) {
+  const bookdeta = props;
+  return (
+    <div className="book-container">
+      <h2>{bookdeta.title}</h2>
+      <h3>{bookdeta.author}</h3>
+    </div>
+  );
```

Comment:
- [ ] As per project requirements, the Book component should have a button for removing books. Can check this out.
![image](https://user-images.githubusercontent.com/62849872/213615982-1208d1da-fc66-48fc-8e50-0486404c9b5e.png)